python/jtmodules/src/jtmodules/combine_masks.py

```diff
@@ -12,36 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 '''Jterator module for combining objects from two binary mask images into one
-by performing an element-wise logical OR operation.
+by performing an element-wise logical operation [AND or OR or XOR].
 '''
 import numpy as np
 import logging
 import collections
 
 logger = logging.getLogger(__name__)
 
-VERSION = '0.0.1'
+VERSION = '0.0.2'
 
 Output = collections.namedtuple('Output', ['combined_mask', 'figure'])
 
 
-def main(mask_1, mask_2, plot=False):
+def main(mask_1, mask_2, logical_operation, plot=False):
     '''Combines two binary masks, such that the resulting combined mask
     is ``True`` where either `mask_1` OR `mask_2` is ``True``.
-
     Parameters
     ----------
-    mask_1: numpy.ndarray[numpy.bool]
-        2D binary array
-    mask_2: numpy.ndarray[numpy.bool]
-        2D binary array
-
+    mask_1: numpy.ndarray[Union[numpy.bool, numpy.int32]]
+        binary or labeled mask
+    mask_2: numpy.ndarray[Union[numpy.bool, numpy.int32]]
+        binary or labeled mask
+    logical_operation: str
+        name of the logical operation to be applied
+        (options: ``{"AND", "OR", "EXCLUSIVE_OR"}``)
     Returns
     -------
     jtmodules.combine_objects.Output
-
     '''
-    combined_mask = np.logical_or(mask_1, mask_2)
+    mask_1 = mask_1 > 0
```

Comment:
I would use `mask_1 != 0` -- I'm not sure booleans are an ordered type (in the case of binary masks).